src/Rules/Generics/GenericAncestorsCheck.php

```diff
@@ -106,6 +110,18 @@ public function check(
 			foreach ($this->varianceCheck->check($variance, $ancestorType, $messageContext) as $message) {
 				$messages[] = $message;
 			}
+
+			foreach ($ancestorType->getVariances() as $index => $typeVariance) {
+				if ($typeVariance->invariant()) {
+					continue;
+				}
+
+				$messages[] = RuleErrorBuilder::message(sprintf(
+					$typeProjectionIsNotAllowedMessage,
+					TypeProjectionHelper::describe($ancestorType->getTypes()[$index], $typeVariance, VerbosityLevel::typeOnly()),
+					$ancestorType->describe(VerbosityLevel::typeOnly()),
+				))->identifier('generics.typeProjectionNotAllowed')->build();
```

Comment:
Now that you want this in `1.10.x`, is adding an identifier here ok, or should I remove it and add it afterwards only to `1.11.x`?

Reply:
This is okay, I might change the identifier though :)